output/aerokube/2.3.7+2/Config.json

```diff
@@ -0,0 +1 @@
       },\n         \"default\": {\n          \"cpu\": \"250m\",\n          \"memory\": \"128Mi\"\n         },\n         \"type\": \"object\"\n        },\n        \"requests\": {\n         \"additionalProperties\": {\n          \"anyOf\": [\n           {\n            \"type\": \"integer\"\n           },\n           {\n            \"type\": \"string\"\n           }\n          ],\n          \"pattern\": \"^(\\\\+|-)?(([0-9]+(\\\\.[0-9]*)?)|(\\\\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\\\\+|-)?(([0-9]+(\\\\.[0-9]*)?)|(\\\\.[0-9]+))))?$\",\n          \"x-kubernetes-int-or-string\": true\n         },\n         \"default\": {\n          \"cpu\": \"100m\",\n          \"memory\": \"128Mi\"\n         },\n         \"type\": \"object\"\n        }\n       },\n       \"type\": \"object\"\n      },\n      \"version\": {\n       \"type\": \"string\"\n      }\n     },\n     \"type\": \"object\"\n    }\n   },\n   \"type\": \"object\"\n  },\n  \"group\": {\n   \"default\": {},\n   \"properties\": {\n    \"id\": {\n     \"default\": 4096,\n     \"format\": \"int64\",\n     \"type\": \"integer\"\n    },\n    \"name\": {\n     \"default\": \"user\",\n     \"pattern\": \"^[a-z_][a-z0-9_-]{0,31}\\\\${0,1}$\",\n     \"type\": \"string\"\n    }\n   },\n   \"type\": \"object\"\n  },\n  \"serviceAccountName\": {\n   \"default\": \"default\",\n   \"pattern\": \"^[a-z0-9]([-a-z0-9]*[a-z0-9])?(\\\\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$\",\n   \"type\": \"string\"\n  },\n  \"sessionTimeout\": {\n   \"default\": \"5m\",\n   \"pattern\": \"^[0-9][0-9]*[smh]$\",\n   \"type\": \"string\"\n  },\n  \"storage\": {\n   \"default\": {},\n   \"properties\": {\n    \"accessKey\": {\n     \"default\": \"\",\n     \"type\": \"string\"\n    },\n    \"bucket\": {\n     \"default\": \"\",\n     \"type\": \"string\"\n    },\n    \"endpoint\": {\n     \"default\": \"\",\n     \"type\": \"string\"\n    },\n    \"filename\": {\n     \"default\": \"\",\n     \"type\": \"string\"\n    },\n    \"pattern\": {\n     \"default\": \"\",\n     \"type\": \"string\"\n    },\n    \"secretKey\": {\n     \"default\": \"\",\n     \"type\": \"string\"\n    },\n    \"secretRef\": {\n     \"default\": null,\n     \"properties\": {\n      \"accessKey\": {\n       \"type\": \"string\"\n      },\n      \"name\": {\n       \"pattern\": \"^[a-z0-9]([-a-z0-9]*[a-z0-9])?(\\\\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$\",\n       \"type\": \"string\"\n      },\n      \"secretKey\": {\n       \"type\": \"string\"\n      }\n     },\n     \"required\": [\n      \"name\"\n     ],\n     \"type\": \"object\"\n    }\n   },\n   \"type\": \"object\"\n  },\n  \"user\": {\n   \"default\": {},\n   \"properties\": {\n    \"id\": {\n     \"default\": 4096,\n     \"format\": \"int64\",\n     \"type\": \"integer\"\n    },\n    \"name\": {\n     \"default\": \"user\",\n     \"pattern\": \"^[a-z_][a-z0-9_-]{0,31}\\\\${0,1}$\",\n     \"type\": \"string\"\n    }\n   },\n   \"type\": \"object\"\n  }\n },\n \"title\": \"Config\",\n \"type\": \"object\"\n}"}
```

Comment:
Is version 2.3.7+2 really a thing? The "+" seems odd. Does the project use this in their versioning?

Reply:
Yes this is what was scraped